_includes/codepen.html

```diff
@@ -82,38 +90,35 @@
 
 {% endif %}
 
-{% capture script_tag %}
```

Comment:
This is used for the tiny drive demos isn't it? eg here: https://github.com/tinymce/tinymce-docs/blame/master/tinydrive/tinydrive-api/standalone.md#L56

Reply:
the TinyDrive script tag is added in https://github.com/tinymce/tinymce-docs/pull/1572/files#diff-67ca7b414090983c68ec730a7b673120R35
Capture creates variables, but this one is never called.